src/ansys/tools/visualization_interface/backends/pyvista/pyvista.py

```diff
@@ -324,8 +315,10 @@ def hover_callback(self, _widget, event_name) -> None:
         renderer = plotter.iren.get_poked_renderer(x, y)
         self._hover_picker.Pick(x, y, 0, renderer)
         actor = self._hover_picker.GetActor()
-        if event_name == "TimerEvent" and actor is not None and actor in self._object_to_actors_map:
```

Comment:
I have to recognaise I didn't understand why the ``"TimerEvent"``, so I am OK to remove it... I guess...

Reply:
`TimerEvent` is the event name you get when you are hovering over an actor (weird name in my opinion but it's what you get). When you are not hovering over an actor, you get `EndInteraction`. I was using both for detecting when you are in or out of an actor

Removing this is kind of a tradeoff. If you use the event names, it looks laggy, but the label disappears the instant you hover out of the actor. Now, it doesn't look laggy, but the label stays a bit longer after hovering out of the actor.

Reply:
I guess that is a race issue ... anyway, small delays to remove the hover is not too bad IMHO.